.travis.yml

```diff
@@ -5,7 +5,7 @@ go:
 
 install:
   - make dependencies
-  - $(exit $(go fmt ./... | wc -l))
+  - test -z "$(go fmt ./...)"
```

Comment:
Can you submit another PR where gofmt hasn't been run, to demonstrate that this command fails properly?